### PR TITLE
Add css var for indeterminate_cycle_duration

### PR DESCRIPTION
**Pull request by lena**
Enable customization of the indeterminate duration cycle 

Demo with 20s duration:
indeterminate fake

### CODE REVIEW

**Comment by lena** [comment]
I signed it!

**Comment by keanulee** [comment]
@lena This looks great! Mind adding a demo to the demo page?

**Comment by lena** [comment]
@keanulee sure thing 👍 will do it today

File: demo/index.html [hunk 1]
```diff
@@ -49,11 +49,16 @@
...
+    <h3>paper-progress can be indeterminate with a custom duration</h3>
     <demo-snippet class="centered-demo">
       <template>
+        <style is="custom-style">
+          paper-progress.slow {
+            --paper-progress-indeterminate-cycle-duration: 20s;
+          }
+        </style>
         <paper-progress indeterminate></paper-progress>
-        <paper-progress indeterminate value="800" min="100" max="1000"></paper-progress>
```
Comment (lena) [inline demo/index.html:56]: these props are ignored by the indeterminate progress bar, so i deleted them

**Comment by lena** [comment]
@keanulee done

**Comment by keanulee** [comment]
LGTM. Thanks @lena!

**Pull request merged**